CHANGELOG.md

```diff
@@ -3,11 +3,30 @@
 ### Added
 
 - Add readiness probe to etcd pod. The readiness state will be reflected on `status.members.ready/unready`.
+- TLS etcd cluster support in backup/restore-operator.
+- Add spec validation in restore operator.
+- Add BackupStorageType to EtcdRestore.RestoreSpec to indicate type of the backup storage which is used as RestoreSource.
+- Add EtcdClusterRef to EtcdRestore.RestoreSpec to reference an EtcdCluster resource which should have "Failed" status.phase and to be restored.
```

Comment:
~~which should have "Failed" status.phase and to be restored.~~ whose metadata and spec will be used to create the new restored EtcdCluster CR.